test/okta_jwt_elixir_test.exs

```diff
@@ -0,0 +1,8 @@
+defmodule OktaJwtElixirTest do
+  use ExUnit.Case
+  doctest OktaJwtElixir
+
+  test "greets the world" do
+    assert OktaJwtElixir.hello() == :world
+  end
```

Comment:
TODO: Add tests, minimally for claims checks. When the HTTP client is created, add Mox to mock requests to okta servers